### DISCUSSION TITLE
How can i use blueprint from cdn with typescipt and webpack?

### DISCUSSION THREAD

**Discussion by no95typem** [palantir/blueprint#5750] [Q&A]
#### Environment

- __Package version(s)__:  Core 3.47
- __Browser and OS versions__: Chromium, Linux

#### Question

Hello! I'm trying to make blueprint external dependency, but I don’t understand how can I import it with typescript and webpack? I can't just write this due to the presence of a point:

```
import * as Blueprint.Core from '@blueprintjs/core'
```

and then add
```
externals: {
  '@blueprintjs/core': 'Blueprint.Core',
}
```
in webpack config.

So I need an example. Could anyone give me one, please?

**Comment by adidahiya** [comment]
Don't use a `.` in the symbol name. Just write it differently, for example as `BlueprintCore`.

**Comment by no95typem** [comment]
OK, but 
```
The UMD wrapper exposes each library on the Blueprint global variable: Blueprint.Core, Blueprint.Datetime, etc.
```
So, BlueprintCore will be undefined in this case. Unfortunately I don't know a method to solve this issue.
